File: LES_EXAMENS/EXAM_PYTHON_NORMAL_2024_2025_MIP/exerciceE.c
#include <stdio.h>
#include <stdbool.h>
int temp,T[10];
bool verifier;
int main(){
    for(int i=0;i<10;i++){
        printf("saisir l'entier numero %d: ",i+1);
        scanf("%d",&T[i]);
    }
    do{
        verifier=true;
        for(int i=0;i<9;i++){
            if(T[i]>T[i+1]){
                temp=T[i];
                T[i]=T[i+1];
                T[i+1]=temp;
                verifier=false;
            }
        }
    }while(verifier==false);
    for(int i=0;i<10;i++){
        if(T[i]!=0){
            printf("%d\n",T[i]);
        }
    }
    return 0;
}

File: LES_EXAMENS/EXAM_PYTHON_NORMAL_2024_2025_MIP/exerciceE.py
T=[None for i in range(10)]
for i in range(0,10):
    T[i]=int(input(f"saisir l'entier numero {i+1}: "))
while True:
    verifier=True
    for i in range(0,9):
        if T[i]>T[i+1]:
            temp=T[i]
            T[i]=T[i+1]
            T[i+1]=temp
            verifier=False
    if verifier==True:
        break
for i in range(0,10):
    if T[i]!=0:
        print(T[i])
